tests/jest/webpack.config.test.js

```diff
@@ -0,0 +1,214 @@
+jest.mock('../../scripts/aggregate-translations/aggregate-translations');
```

Comment:
Does this file need to have the .test extension? Feels like this would always be ran in the test mode, making the extension redundant.

Reply:
Disregard. I realize this is a testing configuration, not an exportable script.

Reply:
yes! This was me mocking the `aggregate-translations` call